### PR TITLE
Add rosco package

### PR DESCRIPTION
**Pull request by nikhar-abbas**
Happy to be a maintainer! Thanks

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rafmudaf** [comment]
@conda-forge-admin, please ping team
I think this recipe is ready to roll. Is there anything else we should do to finish up?

File: recipes/rosco/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+# source:
+#   url: https://github.com/nrel/rosco/archive/v{{ version }}.tar.gz
+#   sha256: 3e907f7d2f8369cb20c08050985e00473176e8b5cde85ebda8f77e8d18135a2f
+#   # Download the tar-ball and do `openssl sha256 file.tar.gz` to update
```
Comment (scopatz) [inline recipes/rosco/meta.yaml:11]: Please remove comments

File: recipes/rosco/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+source:
+  git_url: https://github.com/NREL/ROSCO.git
+  # git_sh: 47a13446063a136c7f22ea09d3569280b62668dd
+  git_rev: master
```
Comment (scopatz) [inline recipes/rosco/meta.yaml:16]: Please build from a source tarball, not git

File: recipes/rosco/meta.yaml (hunk 1 above)
Comment (rafmudaf) [reply recipes/rosco/meta.yaml]: Done!

File: recipes/rosco/meta.yaml (hunk 2 above)
Comment (rafmudaf) [reply recipes/rosco/meta.yaml]: And done!

File: recipes/rosco/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{% set name = "rosco" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nrel/rosco/archive/master.tar.gz
+  sha256: 06e2f01a8cc20d8fe271ad987f58e1996ab5352bd35e69f16a449c182860fcde
```
Comment (scopatz) [inline recipes/rosco/meta.yaml:10]: This is not a released version

Comment (rafmudaf) [reply recipes/rosco/meta.yaml:10]: @nikhar-abbas can you create a new release and update the recipe?

Comment (nikhar-abbas) [reply recipes/rosco/meta.yaml]: Fixed!

File: recipes/rosco/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+{% set version = "2.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nrel/rosco/archive/v{{ version }}.tar.gz
+  sha256: fc4813439be6cbf275cee1fa884e521490559ca87d2f63498ae4fab2dfa9bc48
+  # Download the tar-ball and do `openssl sha256 file.tar.gz` to update
```
Comment (scopatz) [inline recipes/rosco/meta.yaml:11]: Please remove comments

Comment (nikhar-abbas) [reply recipes/rosco/meta.yaml]: removed

**Comment by scopatz** [comment]
The master.tar.gz file should not be checked in.  Can you please close this PR and on a totally separate branch add only the needed files? Thanks!

**Comment by scopatz** [comment]
And then open a new PR

**Pull request closed without merging**